esper/ext/pipeline_api.py

```diff
@@ -0,0 +1,389 @@
+import requests
+
+
+class APIException(Exception):
+    pass
+
+
+def get_pipeline_url(environment: str,
+                     enterprise_id: str,
+                     pipeline_id: str = None) -> str:
+    """
+    Build and return pipeline url for scapi endpoint
+    :param environment:
+    :param enterprise_id:
+    :param pipeline_id:
+    :param stage_id:
+    :param operation_id:
+    :return: Url
+    """
+
+    url = f'https://{environment}-api.esper.cloud/api/v1/enterprise/{enterprise_id}/pipeline/'
+
+    if pipeline_id:
+        url += f'{pipeline_id}/'
+
+    return url
+
+
+def get_pipeline_execute_url(environment: str,
+                             enterprise_id: str,
+                             pipeline_id: str,
+                             execute_id: str = None,
+                             action: str = None) -> str:
+    """
+    Build and return pipeline Execute url for scapi endpoint
+    :param environment:
+    :param enterprise_id:
+    :param pipeline_id:
+    :param execute_id:
+    :param action:
+    :return: Url
+    """
+
+    url = f'https://{environment}-api.esper.cloud/api/v1/enterprise/{enterprise_id}/pipeline/{pipeline_id}/execute/'
```

Comment:
Why are using direct url instead of SDK? It's not available in SDK?

Reply:
not yet